docs/weave-tutorial/06-buckets.md

```diff
@@ -57,64 +61,61 @@ This basically consists of adding _Copy()_ and _Validate()_ to the objects in `c
 
 ## Dive into Code
 
-Let's define `MarketBucket` that will hold `Market` information and write a function that creates a market. This is a basic `morm/model_bucket` without any secondary index.
+Let's define `UserBucket` that will hold `User` information and write a function that creates an user. This is a basic `morm/model_bucket` without any secondary index.
```

Comment:
`Let's define `UserBucket` that will hold `User` information and write a function that creates a user. This is a basic `morm/model_bucket` without any secondary index.
`